tasks/reduce-snapshot/tests/test-reduce-snapshot-wrong-component.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-reduce-snapshot-wrong-component
+spec:
+  description: |
+    Run the reduce task with a snapshot mentioning a component that does not exist
```

Comment:
that does not exist. So, the entire snapshot should be returned instead of it being reduced

Reply:
fixed description